cypher-shell/src/test/java/org/neo4j/shell/test/bolt/FakeSession.java

```diff
@@ -21,24 +29,42 @@ public Transaction beginTransaction(String bookmark) {
         return null;
     }
 
+    @Override
+    public CompletionStage<Transaction> beginTransactionAsync()
+    {
```

Comment:
You're using Neo4j codestyle which is inconsistent with the rest of the code in this project. Just use IntelliJ default